UserAdminBundle/Form/Type/UserType.php

```diff
@@ -108,6 +114,14 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 ));
             $builder->addEventSubscriber($this->userProfileSubscriber);
             $builder->addEventSubscriber($this->userGroupSubscriber);
+            if ($this->allowedToSetPlatformAdmin || $this->allowedToSetDeveloper) {
+                $builder->add('accountLocked', 'checkbox', array(
```

Comment:
Is enabled specific to lock as there are : isCredentialsNonExpired, isAccountNonExpired ...?
In AdvancedUserInterface, you get a isEnabled method and a isAccountNonExpired method. So I'm not sure that using enabled is a good idea.